tests/yam/agama/import_agama_profile.pm

```diff
@@ -0,0 +1,20 @@
+## Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+# Summary: Run Agama profile import on Live Medium
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use base Yam::Agama::patch_agama_base;
+use strict;
+use warnings;
+use Utils::Architectures 'is_s390x';
+use testapi qw(assert_script_run data_url get_required_var select_console);
+
+sub run {
+    select_console 'root-console' unless is_s390x;
```

Comment:
drop is_s390x until you start to verify there.

Reply:
Done